source/camera/CameraManager.h

```diff
@@ -48,6 +51,7 @@ enum class ViewModes : uint16_t
 
 class CameraManager
 {
+    friend class CullingManager;
```

Comment:
Please do not use friendship unless there is a real reason. It makes search and code reading not straight forward. If you need access to some of the CameraManager variables, just add the corresponding getters/setters so that we know at first glance if a private variable is private or not.